web-bluetooth/write-descriptor-async-await.html

```diff
@@ -0,0 +1,50 @@
+---
+feature_name: Web Bluetooth / Characteristic User Description (Async Await)
```

Comment:
update feature name to more clearly communicate that this is about writing to a descriptor.